packages/ckeditor5-dev-tests/bin/save-revision.js

```diff
@@ -63,6 +64,8 @@ if ( exec( 'git status -s' ).trim().length ) {
 
 	const lastCommit = exec( 'git log -1 --format="%h"' );
 	console.log( `Successfully saved the revision under ${ mainRepoUrl }/commit/${ lastCommit }` );
+} else {
+	console.log( 'Nothing to commit.' );
```

Comment:
Something more explanatory would be better. A simple "nothing to commit" doesn't explain what are you looking at (you will see this message in travis logs). Something like this might be better:

> Nothing to commit. Revision log is up to date.